tests/middleWare.ts

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import validateAccessJWT from '../server/validation/validateAccessJWT';
+import validateFile from '../server/validation/validateFile';
+import validateRefreshJWT from '../server/validation/validateRefreshJWT';
+import mockInitDB from './mocks/mockInitDB';
+import proxyquire = require('proxyquire');
+import chai = require('chai');
+import sinon = require('sinon');
+import jwt = require('jsonwebtoken');
+require('dotenv').config();
+const expect = chai.expect;
+const validateUsernameCookie = proxyquire(
+	'../server/validation/validateUsernameCookie',
+	{ '../db/initDB': mockInitDB }
+).default;
+const validateUsernameParam = proxyquire(
+	'../server/validation/validateUsernameParam',
+	{ '../db/initDB': mockInitDB }
+).default;
+const verifyUser = proxyquire('../server/validation/verifyUser', {
+	'../db/initDB': mockInitDB
+}).default;
+
+describe('Unit testing of express middleware', function () {
+	let res;
+	let resSpy;
+	let nextSpy;
+	beforeEach(function () {
+		resSpy = sinon.spy();
+		nextSpy = sinon.spy();
+		res = {
+			sendStatus: resSpy
+		};
+	});
+	it('should validate access token', function () {
+		const req = {
+			headers: {
+				authorization: undefined
+			}
+		};
+		// with no token
+		validateAccessJWT(req, res, () => {});
+		expect(resSpy.calledOnce).to.equal(true);
+		expect(resSpy.args[0][0]).to.equal(401);
+		// with an invalid token
+		req.headers.authorization =
+			'token ' +
+			jwt.sign(
+				{ username: 'testUsername' },
+				process.env.ACCESS_JWT_SECRET + 'INVALID'
+			);
```

Comment:
This should be two different tests. They may be grouped in additional nested `describe`.
And the same for cases in other tests.